src/vws/exceptions.py

```diff
@@ -18,80 +40,33 @@ class UnknownVWSErrorPossiblyBadName(Exception):
     """
 
 
-class ConnectionErrorPossiblyImageTooLarge(
-    requests.exceptions.ConnectionError,
```

Comment:
Was there value derived from inheriting from `requests.exceptions.ConnectionError` which is now lost?

Reply:
No, you [raised the `ConnectionErrorPossiblyImageTooLarge` class](https://github.com/VWS-Python/vws-python/blob/master/src/vws/query.py#L125), so Python instantiates it without any arguments. If you want to retain the request and response object in the exception, you would have to do:

```py
raise ConnectionErrorPossiblyImageTooLarge(request=exc.request, response=exc.response) from exc
```